conans/client/rest/rest_client_common.py

```diff
@@ -196,7 +196,7 @@ def get_json(self, url, data=None, headers=None):
 
         content = decode_text(response.content)
         content_type = response.headers.get("Content-Type")
-        if content_type != 'application/json':
+        if content_type != 'application/json' and content_type != 'application/json; charset=utf-8':
```

Comment:
looks good.
in future, we may consider accepting more encodings there, according to RFC:
https://tools.ietf.org/html/rfc4627
https://tools.ietf.org/html/rfc7158#section-8.1